src/components/embed/ui/molecules/card/auth-request-card/AuthRequestCard.tsx

```diff
@@ -20,10 +21,24 @@ export function AuthRequestCard({
   onConfirm,
   cancelLabel = browser.i18n.getMessage("cancel"),
   confirmLabel = browser.i18n.getMessage("continue"),
-  isConfirmDisabled,
+  areButtonsDisabled,
   children: childrenProp,
   ...cardProps
 }: AuthRequestCardProps) {
+
+  useEffect(() => {
+    function handleKeyDown(e: KeyboardEvent) {
```

Comment:
This functionality was duplicated in most (but not all) auth request views before.